docs/examples/app-the-new-thing/src/interbit/control/constants.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
+  CHAIN_ALIASES: {
+    CONTROL: 'templateControl',
+    PUBLIC: 'templatePublic',
+    PRIVATE: 'templatePrivate'
+  },
+  COVENANTS: {
+    PRIVATE: 'template-private'
+  }
+}
```

Comment:
Please change chain and covenant aliases to avoid naming collisions. This is IMPORTANT. Make sure it is documented too.